istioctl/pkg/util/handlers/handlers.go

```diff
@@ -135,23 +135,29 @@ func getClientForResource(resname, ns string, factory cmdutil.Factory) (*corev1c
 	return clientset, "", namespace, selector.String(), nil
 }
 
+var getFirstPodFunc func(client corev1client.PodsGetter, namespace string, selector string, timeout time.Duration,
```

Comment:
What's the benefit to define this proto function?

Reply:
This is only for testing purposes. There's no existing test that covers this, and without defining the function, it's hard to write a test for the entire function.

Reply:
add comment like https://github.com/istio/istio/blob/98c4f535dea16ddfa1fe04fe8a28c8f510edd0a1/pilot/pkg/networking/core/v1alpha3/tracing.go#L58

Reply:
okay, got it. Please add the comment for it.

Reply:
Thanks, done.